src/wrterm.h

```diff
@@ -0,0 +1,160 @@
+#ifndef __WRTERM_H_
+#define __WRTERM_H_
+
+#include "dispextern.h"
+
+struct wr_bitmap_record
+{
+  char *file;
+  int refcount;
+  int height, width, depth;
+};
+
+typedef int Screen;
+
+struct wr_display_info
+{
+  /* Chain of all wr_display_info structures.  */
+  struct wr_display_info *next;
+
+  /* The generic display parameters corresponding to this NS display.  */
+  struct terminal *terminal;
+
+  /* This is a cons cell of the form (NAME . FONT-LIST-CACHE).  */
+  Lisp_Object name_list_element;
```

Comment:
Are there any concerns for GC tracing of `name_list_element` here? Or do we expect the currently existing codebase to handle this?

Reply:
The existing codebase needs this field in `xxx_display_info` struct. I think they can handle it properly.